templates/provider/hmc/templates/webhooks.yaml

```diff
@@ -127,6 +127,29 @@ webhooks:
         resources:
           - clustertemplates
     sideEffects: None
+  - admissionReviewVersions:
+      - v1
+      - v1beta1
+    clientConfig:
+      service:
+        name: {{ include "hmc.webhook.serviceName" . }}
+        namespace: {{ include "hmc.webhook.serviceNamespace" . }}
+        path: /validate-hmc-mirantis-com-v1alpha1-servicetemplate
+    failurePolicy: Fail
+    matchPolicy: Equivalent
+    name: validation.servicetemplate.hmc.mirantis.com
+    rules:
+      - apiGroups:
+          - hmc.mirantis.com
+        apiVersions:
+          - v1alpha1
+        operations:
+          - CREATE
```

Comment:
please leave only DELETE as we don't have anything else implemented yet

Reply:
I should probably remove CREATE and UPDATE from clustertemplate as well because that also only has validation for DELETE implemented.